src/System.Runtime.InteropServices.RuntimeInformation/ref/System.Runtime.InteropServices.RuntimeInformation.cs

```diff
@@ -21,8 +21,20 @@ public partial struct OSPlatform : System.IEquatable<System.Runtime.InteropServi
         public static bool operator !=(System.Runtime.InteropServices.OSPlatform left, System.Runtime.InteropServices.OSPlatform right) { return default(bool); }
         public override string ToString() { return default(string); }
     }
+
+    public enum Architecture
+    {
+        x86,
+        x64,
+        ARM
+    }
+
     public static partial class RuntimeInformation
     {
+        public static Architecture GetProcessArchitecture() { return default(Architecture); }
+        public static Architecture GetOSArchitecture() { return default(Architecture); }
+        public static string GetOSDebugName() { return default(string); }
```

Comment:
It isn't about protecting, as if that were the case we would still be having a conversation of whether or not to have it at all (which I'm still also hesitant about but I've conceded on). It is about conveying the intent, I don't think DisplayName conveys the intent as it is more than just the name. I'm not hung up on calling it DebugInfo either but I think that at least makes people think about what its purpose is before using it and allows them to realize it contains a more information then just the name.